### PR TITLE
add single product view

### PR DESCRIPTION
**Pull request by miaolphoebe**
Screen Shot 2022-11-08 at 5 35 20 PM

### CODE REVIEW

File: client/store/index.js [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
+import { createLogger } from "redux-logger";
+import thunkMiddleware from "redux-thunk";
+import { composeWithDevTools } from "redux-devtools-extension";
+import auth from "./auth";
+import singleProductReducer from "./singleProduct";
 
-const reducer = combineReducers({ auth })
+const reducer = combineReducers({
+  auth,
+  singleProduct: singleProductReducer,
```
Comment (MichelleMullane) [inline client/store/index.js:10]: I just want to flag this in case others add their reducers differently.

File: server/api/products.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+const router = require("express").Router();
+const {
+  models: { Product },
+} = require("../db/index");
+
+router.get("/", async (req, res, next) => {
```
Comment (MichelleMullane) [inline server/api/products.js]: I already wrote this same route, so this may be a conflict

**Review by MichelleMullane — COMMENTED** [review]
I added some comments.

File: server/api/products.js (hunk 2 above)
Comment (miaolphoebe) [reply server/api/products.js]: I will delete this on my side, and it won't conflict with yours.

**Comment by MichelleMullane** [comment]
All good!  Looks great!

**Review by MichelleMullane — APPROVED** [review]
Looks good!

**Pull request merged**